python/ray/serve/deployment_state.py

```diff
@@ -330,6 +367,7 @@ def check_ready(self) -> Tuple[ReplicaStartupStatus, Optional[DeploymentVersion]
                 )
                 self._health_check_period_s = deployment_config.health_check_period_s
                 self._health_check_timeout_s = deployment_config.health_check_timeout_s
+                self._node_id = ray.get(self._allocated_obj_ref)
```

Comment:
What is allocated_obj_ref and why does it make sense to use it as a node ID? Specifically why do two replicas have the same `ray.get(self._allocaterd_obj_ref)` when on the same node?  I think it would be worth adding a comment either here or in the docstring for `self._allocated_obj_ref`

Reply:
Oh I see, it calls is_allocated which returns the node ID from Ray.  I guess that's straightforward enough to not need a comment